lib/Refactoring/ExtractExprBase.h

```diff
@@ -47,6 +47,9 @@ class ExtractCheckResult {
   }
 };
 
+/// Check whether a given range can be extracted.
+/// Return true on successful condition checking,.
+/// Return false on failed conditions.
```

Comment:
Was this extract from somewhere or new? It doesn't actually return a bool

Reply:
Came from here

https://github.com/apple/swift/blob/df4f824b02947ca017f3c722d7803d222e52830f/lib/Refactoring/Refactoring.cpp#L1116-L1118

Reply:
:(